parsers/mirex05melody_parser.py

```diff
@@ -80,14 +85,15 @@ def process_folder(in_dir, out_dir):
 
     # Collect all melody f0 annotations.
     f0_files = list()
```

Comment:
note sure, this code was already there (and I don't think I wrote it). Anyway, it's changed: fad8ab2